app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinPresenter.kt

```diff
@@ -22,6 +21,7 @@ class NewCreatePinPresenter(
 
     private var createdPin = emptyString()
     private var pinMode = PinMode.CREATE
+    private var isOnBackButtonPressed = false
```

Comment:
isOnBackButtonPressed  -> navigateBackOnBackPressed
we need to put some logic behind this name that is connected with exiting app when back is pressed twice